webapp/src/main/webapp/src/app/components/label-list/label-list.component.ts

```diff
@@ -18,17 +18,17 @@ import { Component, OnInit, ViewEncapsulation, ChangeDetectionStrategy, Input} f
 import { Metadata } from '../../../app/models/commons.model';
 
 @Component({
-  selector: 'label-list',
+  selector: 'app-label-list',
   encapsulation: ViewEncapsulation.None,
   templateUrl: './label-list.component.html',
   styleUrls: ['./label-list.component.css'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class LabelListComponent implements OnInit {
 
-  @Input('metadata') metadata: Metadata
+  @Input() metadata: Metadata;
 
-  @Input('filter') filter: string
+  @Input() filter: string;
```

Comment:
Is it safe to remove the name of the Input?

Reply:
Yes, it's a general approach to avid specifying name in a decorator